scanpy/tests/blackdiff.py

```diff
@@ -45,8 +45,9 @@ def black_diff(src: Path, differ=Differ()) -> Tuple[int, int]:
     ]
     for file, (added, removed) in sorted(excluded, key=lambda sd: -sum(sd[1])):
         print(f'{file}: +{added} -{removed}')
-        if added + removed < thresh:
-            print('File has < 10 changes to black formatting. Do it!')
+        if 0 < added + removed < thresh:
+            print(f'File: {file}: has < {threshold} changes to '
+                  f'black formatting. Do it!')
```

Comment:
No, if it’s 0, it should be removed from `tool.black.exclude` in `pyproject.toml`.